src/OrchardCore.Modules/OrchardCore.Localization/RequestLocalizationOptionsExtensions.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Localization;
+
+namespace OrchardCore.Localization
+{
+    internal static class RequestLocalizationOptionsExtensions
+    {
+        public static RequestLocalizationOptions SetDefaultCulture(
+            this RequestLocalizationOptions options,
+            bool useUserOverride,
+            string defaultCulture)
+        {
+            options.DefaultRequestCulture = new RequestCulture(new CultureInfo(defaultCulture, useUserOverride));
+
+            return options;
+        }
+
+        public static RequestLocalizationOptions AddSupportedCultures(
+            this RequestLocalizationOptions options,
+            bool useUserOverride,
```

Comment:
I expect you will add `useUserOverride` as first param, but it's weird from API design perspective

No problem if it's hosted in the module itself

Reply:
`useUserOverride` can't be the first param which is `this options` on which the extension is applied, and can't be the last param because the `string[] params` should be at the end.

Reply:
Yes I know, that's one reason, which change my mind at the beginning  - before introducing `OrchardCoreRequestLocalizationOptions` - when you suggested the extensions

Reply:
Ah okay, but that's okay if internal to the module and only temporary ;)

Reply:
> No problem if it's hosted in the module itself

That's what I intended here ;)